guides/user/pipeline/managing-pipelines/index.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: single
+title:  "Managing Pipelines"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+Pipelines are the essential tool in Spinnaker for controlling how to deploy your
+application. A pipeline is composed of a series of stages that can be combined
+in almost any order, which makes pipelines flexible, consistent, and repeatable.
+
+You can configure your pipelines to run entirely automatically or require manual
+intervention to ensure that everything is working as expected. Similarly,
+pipelines can either be automatically triggered by a wide range of external
+inputs, including other pipelines, or be manually triggered.
+
+This guide explains how to configure and control pipelines, including creating,
+adding triggers to, or disabling pipelines.
+
+## Create a pipeline
+1. Select the application in which you would like to create a pipeline.
+2. Navigate to the **Pipelines** tab in the Spinnaker UI.
+  ![](images/pipelines-tab.png)
+3. Click **Create**, located in the upper right corner of the **Pipelines** tab.
+  ![](images/create.png)
+4. Choose **Pipeline** from the drop down menu and name your pipeline.
+
+After you create your pipeline, add stages to specify the actions that your
+pipeline will perform.
+
+### Add a stage
+The first step in any pipeline is **Configuration**, where you can set up
+pipeline triggers and parameters. Each *stage* specifies an action the pipeline
+will take once it is configured.
+
+1. Select **Add stage** from your pipeline configuration screen.
+  ![](images/add-stage.png)
+2. Set the stage type using the drop-down menu.
+3. If this is not the first stage in your pipeline, make sure that this stage
+depends on the desired upstream stage(s) using the **Depends on** field.
+  ![](images/stage-depends-on.png)
+
+You can add as many stages as your pipeline needs, in any order that makes sense
+for you.
+
+### Add a trigger
+
+Make sure that you are editing the **Configuration** stage of your pipeline.
+
+![](images/configuration-stage.png)
+
+1. Select **Add trigger**.
+  ![](images/add-trigger.png)
+2. Choose your desired trigger type from the drop-down menu that appears, and
+input any further required configuration.
+
+For further information on how triggers work, see the documentation on pipeline
+triggers.
+<!-- TODO(nhayes): link here to pipeline triggers overview once it exists. -->
+
+## Disable a pipeline
+
+Disabling a pipeline prevents any triggers from firing, as well as preventing
+users from running it manually.
+
+1. From the **Pipelines** tab, click **Configure** to modify an existing
+pipeline.
+  ![](images/select-configure.png)
+2. Click **Pipeline actions** in the upper right corner, and select **Disable**.
+  ![](images/pipeline-actions.png)
+
+In order to re-enable your disabled pipeline, select **Pipeline actions** and
+choose **Enable**.
+
+## Delete a pipeline
```

Comment:
If we are not documenting all the actions (I think we're not documenting "rename", for example), is there a specific reason for including this? "Delete" seems fairly self-explanatory, and since this section has screenshots that may need to be updated it adds to our potential maintenance load?

Reply:
I mostly included it because I figured that people might reasonably google 'delete spinnaker pipeline' and I wanted it to show up. FWIW, it reuses screenshots that are also used in explaining other actions, so it doesn't actually add to the maintenance burden.

Reply:
> because I figured that people might reasonably google 'delete spinnaker pipeline' and I wanted it to show up

Ah, clear; thanks for explaining!